data/modules/TradeShips/Trader.lua

```diff
@@ -293,6 +306,39 @@ local trader_task = {}
 -- the task function prototype should be:
 -- function(ship)
 
+trader_task.hyperjumpAtDistance = function(ship)
+	-- the player may have left the system
+	if ship:exists() then
+		local trader = Core.ships[ship]
+		if trader then
```

Comment:
This is not super important IMHO, but you can get rid of 2 indentations if you exit using the opposite condition, i.e. `if not trader then return end`

Reply:
Done